specification/contosowidgetmanager-new2/resource-manager/Microsoft.Contoso/Contoso.Management/employee.tsp

```diff
@@ -0,0 +1,63 @@
+import "@typespec/rest";
```

Comment:
Please rename Contoso.Management folder to just Contoso. We already know it's management because it's under resource-manager

Reply:
Wess requested a data-plane example to be part of this PR.
So, under data-plane folder, there should 1 or 2 ServiceName folders identical in structure to that under the Microsoft.Contoso RP namespace folder. Data-plane does not have an RP namespace folder and does not have a readme.md files in this folder. The only readme.md files are in the ServiceName folders.